manager/src/main/resources/application.properties

```diff
@@ -1,3 +1,5 @@
+event-bridge.k8s.orchestrator=${EVENT_BRIDGE_K8S_ORCHESTRATOR:minikube}
```

Comment:
@r00ta I'd suggest we default to `openshift` so that the service will always behave as if it's running in production unless you configure it otherwise.